packages/cli/src/index.ts

```diff
@@ -793,7 +789,7 @@ const handleUnexpected = async (err: Error) => {
 
   // We do not want to render errors about Sentry not being reachable
   if (message.includes('sentry') && message.includes('ENOTFOUND')) {
-    debug(`Sentry is not reachable: ${err}`);
+    output.error(`Sentry is not reachable: ${err}`);
```

Comment:
>  // We do not want to render errors about Sentry not being reachable

Should we use `output.debug` here?

Reply:
I made a judgement call. I think we should show that even when debug is off. Happy to switch it if others disagree.

Reply:
I don't even know what Sentry is used for so not gonna argue with that 👍